worlds/hylics2/__init__.py

```diff
@@ -88,6 +89,22 @@ def generate_basic(self):
             for i, data in Items.party_item_table.items():
                 pool.append(self.add_item(data["name"], data["classification"], i))
 
+        # handle gesture shuffle
+        if self.multiworld.gesture_shuffle[self.player] == 0: # add gestures to pool like normal
```

Comment:
This isn't bad necessarily, but in python 0 evaluates as false so this can just be `not gesture_shuffle` which reads nicer.